schema/me/index.js

```diff
@@ -78,16 +80,18 @@ export default {
       "sale_registrations",
       "conversation",
       "conversations",
+      "statuses",
       "collector_profile",
       "artwork_inquiries_connection",
       "notifications_connection",
     ]
+    // TODO: Inject fieldNodes as rootFieldNodes
     if (queriedForFieldsOtherThanBlacklisted(fieldNodes, blacklistedFields)) {
       return gravity.with(accessToken)("me").catch(() => null)
     }
 
     // The email and is_collector are here so that the type system's `isTypeOf`
     // resolves correctly when we're skipping gravity data
-    return { id: userID, email: null, is_collector: null }
+    return { rootFieldNodes: fieldNodes, id: userID, email: null, is_collector: null }
```

Comment:
@alloy so basically, from `me { ... }`, we want to be able to inject what is being requested (so within `statuses` we can check if we can reuse anything).